cpp/src/IceSSL/OpenSSLTransceiverI.cpp

```diff
@@ -458,7 +400,7 @@ OpenSSL::TransceiverI::initialize(IceInternal::Buffer& readBuffer, IceInternal::
         out << "SSL summary for " << (_incoming ? "incoming" : "outgoing") << " connection\n";
 
         //
-        // The const_cast is necesary because Solaris still uses OpenSSL 0.9.7.
+        // The const_cast is necessary because Solaris still uses OpenSSL 0.9.7.
```

Comment:
Should we open an issue to get rid of this const cast?

Reply:
We should remove all Solaris CC -related code!